common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -87,6 +87,39 @@ public Double getValue(FunctionArguments arguments) {
         }
     }
 
+    public static class StatusEffectsFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            List<String> statusEffectsList = Models.StatusEffect.getStatusEffects().stream()
+                    .map(statusEffect -> statusEffect.asString().getString())
+                    .toList();
+
+            return String.join("\n", statusEffectsList);
+        }
+    }
+
+    public static class ContainsStatusEffectFunction extends Function<Boolean> {
```

Comment:
Perhaps rename this to "StatusEffectActive"? Don't forget to rename the localization keys.

Reply:
Queued